pkg/router/template/router.go

```diff
@@ -218,6 +218,8 @@ func matchPattern(pattern, s string) bool {
 	return false
 }
 
+// genSubdomainWildcardRegexp is now legacy and around for backward
+// compatibility and allows old templates to continue running.
```

Comment:
if you use config maps, you will run into issues once you upgrade/the latest image changes underneath you.